src/main/webapp/resources/js/components/alerts/ErrorAlert.tsx

```diff
@@ -6,6 +6,11 @@ import React from "react";
 import { Alert } from "antd";
 import { AlertProps } from "antd/lib/alert";
 
+export interface ErrorAlertProps {
```

Comment:
Sorry to point this out now, but this is actually wrong.  You do not need to create this interface at all as `AlertProps` encompasses everything you need.

Reply:
Thanks for pointing this out. As discussed I kept the wrong code when merging in dev. Updated in 152ba864e8160e1aec8276f84e433db80e1d2f1b

Reply:
Beauty, thanks :)